packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -294,15 +299,36 @@ export class AtomicInsightSearchBox {
     return this.bindings.i18n.t('search-box-with-suggestions');
   }
 
-  private onFocus() {
+  private async onFocus() {
     this.isExpanded = true;
-    this.suggestionManager.triggerSuggestions();
+    await this.suggestionManager.triggerSuggestions();
+    this.announceNewSuggestionsToScreenReader();
   }
 
-  private onInput(value: string) {
+  private async onInput(value: string) {
     this.searchBox.updateText(value);
     this.isExpanded = true;
-    this.suggestionManager.triggerSuggestions();
+    await this.suggestionManager.triggerSuggestions();
+    this.announceNewSuggestionsToScreenReader();
+  }
+
+  private announceNewActiveSuggestionToScreenReader() {
+    const ariaLabel = this.suggestionManager.activeDescendantElement?.ariaLabel;
+    if (isMacOS() && ariaLabel) {
+      this.suggestionsAriaMessage = ariaLabel;
+    }
+  }
+
+  private announceNewSuggestionsToScreenReader() {
+    const elsLength =
```

Comment:
els?

Reply:
That was old code I blindly reused / copy pasted; It's for "elementsLength"; I'll rename